wallet/client.go

```diff
@@ -102,7 +103,10 @@ func (c *grpcClient) getTransaction(id tx.ID) (*pactus.GetTransactionResponse, e
 
 func (c *grpcClient) getFee(amount int64, payloadType payload.Type) (int64, error) {
 	res, err := c.transactionClient.CalculateFee(c.ctx,
-		&pactus.CalculateFeeRequest{Amount: amount, PayloadType: pactus.PayloadType(payloadType)})
+		&pactus.CalculateFeeRequest{
+			Amount:      util.ChangeToCoin(amount),
```

Comment:
I think we should use CoinToChainge here.

Reply:
Do we have test for this case?

Reply:
which case? where?

Reply:
getFee?

Reply:
as I can remember it had tests. in the execute module and state module. I need to check one more time.